docs-source/content/userguide/experimental/istio.md

```diff
@@ -89,52 +79,85 @@ metadata:
-to your domain in order to satisy Istio's requirements:
+Istio enforces a number of requirements on pods.  When you enable Istio support in the domain resource, the
+introspect job will automatcially create configuration overrrides with the necessary channels for the domain to satisy Istio's requirements:
 
 * On the Administration Server:
-    * Create a channel called `istio-probe` with listen address `127.0.0.1:8888` (or
+    * A network channel called `istio-probe` with listen address `127.0.0.1:8888` (or
       the port you specified in the `readinessPort` setting).
-    * Create a channel called `istio-t3` with listen address `127.0.0.1` and the port
+    * A network channel called `istio-t3` with listen address `127.0.0.1` and the port
       you specified as the admin port.
-    * Create a channel called `istio-ldap` with listen address `127.0.0.1` and the port
+    * A channel called `istio-ldap` with listen address `127.0.0.1` and the port
       you specified as the admin port, with only the LDAP protocol enabled.
-    * Create a channel called `istio-T3Channel` with listen
-      address `127.0.0.1` and the port you specified as the T3 port.
+    * The introspect job will not create any configuration network channel for external access for you.  You can create a channel called `istio-T3Channel` with listen address `127.0.0.1` and the port you specified as the T3 port in your regular WebLogic domain configuration.
 * In the server template that is used to create Managed Servers in clusters:
-    * Create a channel called `istio-probe` with listen address `127.0.0.1:8888` (or
+    * A channel called `istio-probe` with listen address `127.0.0.1:8888` (or
       the port you specified in the `readinessPort` setting) and the public address
       set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-t3` with listen address `127.0.0.1` and the port
+    * A channel called `istio-t3` with listen address `127.0.0.1` and the port
       you specified as the admin port and the public address
       set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-cluster` with listen address `127.0.0.1` and the port
+    * A channel called `istio-cluster` with listen address `127.0.0.1` and the port
       you specified as the admin port, with only the CLUSTER_BROADCAST protocol enabled,
       and the public address set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-http` with listen address `127.0.0.1:31111` and the
+    * A channel called `istio-http` with listen address `127.0.0.1:31111` and the
       public address set to the Kubernetes Service for the Managed Server. Note that `31111`
-      is the Istio proxy (envoy) port.
-* The create domain job will be configured to disable injection of the Istio sidecar.
+      is the Istio proxy (envoy) port.  You can set it to according to your environment of omit it to use the default (31111)
```

Comment:
You can set it to according to your environment of omit it to use the default (31111) -> You can set it according to your environment or omit it, to use the default (31111).